testsuite/integration-arquillian/tests/pom.xml

```diff
@@ -1600,6 +1600,10 @@
                             <groupId>com.google.guava</groupId>
                             <artifactId>guava</artifactId>
                         </exclusion>
+                        <exclusion>
+                            <groupId>org.codehaus.groovy</groupId>
```

Comment:
Is this intended?

Reply:
It should be probably sent as a separate PR. There is version collision with `groovy-all` artifact in base testsuite when running with `auth-server-quarkus-embedded`.

Version `4.0.16` via `keycloak-juni5` and another is `2.4.10` brought by `creaper-commands`.

I'll create another issue and send a separate PR with it.

Reply:
https://github.com/keycloak/keycloak/issues/28555